### PR TITLE
Tutorial.rst: Add hyperlinks

### PR DESCRIPTION
**Pull request by abhishekbundela**
Add the hyperlink to globs, coalib/results/results_actions,
coalib/output.
Closes coala#113

### CODE REVIEW

**Comment by Nosferatul** [comment]
`Fixes` -> `Closes`

**Comment by abhishekbundela** [comment]
Did that @Nosferatul .

**Comment by arjunsinghy96** [comment]
@abhishekbundela can you please rebase.

**Comment by abhishekbundela** [comment]
Please Merge PR.

File: Users/Tutorial.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ Command Line Interface
 
 In order to specify the files to analyze, you can use the ``--files``
 argument of coala like demonstrated below. For all file paths, you can
-specify (recursive) globs.
+specify (recursive) `globs <../Users/Glob_Patterns.html>`__.
```
Comment (aptrishu) [inline Users/Tutorial.rst]: Please refer the address to http://docs.coala.io/en/latest/Users/Glob_Patterns.html .

**Comment by aptrishu** [comment]
ack 0a908a8

**Comment by Mixih** [comment]
@rultor merge

**Pull request merged**